plugin/evm/validators_transitioner.go

```diff
@@ -0,0 +1,138 @@
+// (c) 2019-2021, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+
+	"github.com/flare-foundation/flare/ids"
+	lru "github.com/hashicorp/golang-lru"
+)
+
+// ValidatorsTransitioner transitions validators from a static set of validators
+// to a growing set of dynamic validators over a number of smooth steps.
+type ValidatorsTransitioner struct {
+	validators map[ids.ShortID]uint64
```

Comment:
The current nomenclature can be confusing as one can get confused among `validators` here (which is the default one), `FTSO validators`, `providers` and `FTSO providers`. Comments explaining more details of these fields here can be helpful if there aren't better ways to name them.